crates/fiber/Cargo.toml

```diff
@@ -33,3 +33,10 @@ wasmtime-versioned-export-macros = { workspace = true }
 
 [dev-dependencies]
 backtrace = "0.3.68"
+
+[features]
+default = []
```

Comment:
It's ok to drop this if it's empty, and due to this being an internal crate I think it's reasonable to not have default features here anyway.

Reply:
Done!